analytics.js

```diff
@@ -140,6 +141,17 @@ class Analytics {
     }
   }
 
+  callCookieConsentManager(response, destConfig) {
+    const sourceConfig = response.source.config;
+    let cookieConsentInstance;
+    // if (sourceConfig.cookieConsentManager.oneTrust.enabled) {
+    const cookieConsentClass = cookieConsent.OneTrust;
+    cookieConsentInstance = new cookieConsentClass(sourceConfig, destConfig);
+    const enable = cookieConsentInstance.init();
+    return enable;
+    // }
+  }
+
```

Comment:
`analytics.js` should not have any knowledge of `OneTrust` at all

Reply:
done